apps/condo/domains/miniapp/components/AppDescription/PageContent.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { useEffect, useState } from 'react'
+import { Row, Col } from 'antd'
+import type { RowProps, ColProps } from 'antd'
+import get from 'lodash/get'
+import { useIntl } from '@open-condo/next/intl'
+import { Markdown, Carousel, Typography } from '@open-condo/ui'
+import { useOrganization } from '@open-condo/next/organization'
+import { useLazyQuery } from '@open-condo/next/apollo'
+import { MiniAppOutput } from '@app/condo/schema'
+import { ALL_MINI_APPS_QUERY } from '@condo/domains/miniapp/gql'
+import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
+import { useContainerSize } from '@condo/domains/common/hooks/useContainerSize'
+
+import { TopCard } from './TopCard'
+import { DeveloperCard } from './DeveloperCard'
+import { AppCard, MIN_CARD_WIDTH } from '../AppCard'
```

Comment:
Absolute import)

Reply:
Our current code-style agreement allows us to use relative imports inside component Group

Reply:
No you don't) You're allowed to use relative inside module, as you say. So 14 and 15 lines are representing this rule. It's a little bit struggling, that you first of all should look at files tree before you actually can understand destination of imported file

Reply:
Let's imagine I have a structure like this:
```
.
├── ResuablePart.tsx
├── ComplexComponent.tsx
└── EvenMoreComplexComponent/
    ├── index.ts
    └── SomeSpecificPartOfComponent
```

So why then I'm allowed to call `./ReusablePart` from `ComplexComponent` and not allowed to use it in `EvenMoreComplexComponent `.

What I'm trying to explain, that some of components are more than 1 file, but they're in 99.9% flat file list in some folder. So if I put `EvenMoreComplexComponent` files in top level, this will be `./` local import same as in `ComplexComponent, which is currently allowed.

So, here we should either ban ANY relative import, or deal with this cases specifically.

Reply:
Yes, that's how it is. But what if we had the following structure:
```
.
├── ResuableDomainSpecificPart.tsx
├── ComplexComponent.tsx
└── EvenMoreComplexComponent/
    ├── index.ts
    └── SomeSpecificPartOfComponent
    └── SuperSpecificComponent/
        ├── index.ts
        └── SuperSpecificPartOfComponent
```

So based on your logic, we allowed to use `../../ReusableDomainSpecificPart` inside `SuperSpecificComponent`. So the main idea, as far as I can tell was place local reusable part's of the project near to the place of usage. But if you need to split this grouping by another level it's become not so clear to understand what does it refer to.

Anyway your position make sense, but I would say that such nesting in the structure of components should be avoided